src/components/SearchResults/SearchResult.tsx

```diff
@@ -26,19 +22,6 @@ interface SearchResultProps {
  * The SearchResult component displays a single search result element.
  */
 const SearchResult = ({ bib }: SearchResultProps) => {
-  const { isLargerThanLarge: isDesktop } = useNYPLBreakpoints()
```

Comment:
Great, if the app can rely less on `useNYPLBreakpoints` then there'll be fewer flashes when the desktop layout renders.